README.md

```diff
@@ -246,6 +246,109 @@ Or if you use the Gradle wrapper provided by your Gradle project build:
                       -Prun.port=8080
 ```
 
+## Running a function with Pub/Sub Emulator
```

Comment:
Thanks!
Given the content is instructional for a different repo:

github.com/googleapis/java-pubsub

...we may want to add documentation to that repo rather than here. If that repo changes, we don't want to have to update this repo's docs as well. A link may be better.

Or we may want to include this sample here:
https://github.com/GoogleCloudPlatform/java-docs-samples/

We might want to consider putting this documentation on CloudSite rather than GitHub as well.